doc/tech/master.md

```diff
@@ -0,0 +1,99 @@
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+
+<a name="2.1"></a>
+## 2.1 Specification Types
+
+- **Reference Model**: focuses on the __**NFVI Abstraction**__ and how NFVI services and resources are exposed to VNFs. It written at a high enough level that as new **Reference Architectures** and **Reference Implementations** are added, the model document should require few or no changes. Additionally, the Reference Model is intended to be neutral towards VMs or Containers.
+- **Reference Architecture**: High level NFVI system components and their interactions that takes the Reference Model goals and maps them to something that has components in the real world.  It is expected that at least one, but not more than a few, Reference Architecture will conform to the Reference Model.  The intention whenever possible to use existing elements, rather than specify entirely new architectures in support of the high-level goals specified in the **Reference Model**.
+- **Reference Implementation**: Builds on the requirements and specifications developed in the other two documents and adds details so that it can be implemented.  Each Reference Architecture is expected to be implemented by at least one Reference Implementation.
+
+Below is a diagram of the different artifacts that will need to be created to support the implementation of the abstract concepts presented in the **Reference Model**, which are then applied to create the **Reference Architecture**, that will be deployed using the requirements spelled out in the **Reference Implementation**.
+
+<p align="center"><img src="./figures/tech_scope_3.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
+
+<a name="3.0"></a>
+## 3. Principles
+
+Any specification work created within CNTT **must** obey to set of principles specified by CNTT:
+
+[Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+[Reference Architecture Principles](../ref_arch#principles)
+
+<a name="4.0"></a>
+## 4. Technology Direction
+
+<a name="4.1"></a>
+### 4.1 SR-IOV
+
+SR-IOV is a technology that allows 
+
+<a name="4.2"></a>
+### 4.2 PCI-Pass Through
+
+<a name="4.3"></a>
+### 4.3 SmartNICs
+
+<a name="4.4"></a>
+### 4.4 Acceleration Cards
+
+<a name="4.5"></a>
+### 4.5 FPGAs
+
+<a name="4.6"></a>
+### 4.6 GPUs/NPUs
+
+<a name="4.7"></a>
+### 4.7 EPA/NFD
+
+<a name="5.0"></a>
+## 5. Transitional Plans
+
+It is vitally important for CNTT to have working NFVI solutions from NFVI vendors and mature VNFs/CNFs designs from application vendors that is compliant to CNTT specifications. It is also understood that, in some areas, the industry might not have solutions that are fully aligned with CNTT requirements. Therefore, it is important to have transitional plans to move from what is available today in the industry into fully CNTT compliant solutions and designs.
```

Comment:
As above - can we use a term that allows for containerised, bare metal as well as virtualised, instead of NFVI?

> It is vitally important for CNTT to have working solutions from infrastructure vendors ...

Reply:
resolved in: https://github.com/cntt-n/CNTT/pull/681/commits/fbda59df9121b63e20cfc67fabb7aa1e438b0dce